app/screens/settings/notification_auto_responder/notification_auto_responder.tsx

```diff
@@ -88,8 +88,9 @@ const NotificationAutoResponder = ({currentUser, componentId}: NotificationAutoR
                 auto_responder_message: autoResponderMessage,
             },
         });
+        fetchStatusInBatch(serverUrl, currentUser.id);
         close();
-    }, [serverUrl, autoResponderActive, autoResponderMessage, notifyProps]);
+    }, [serverUrl, autoResponderActive, autoResponderMessage, notifyProps, currentUser]);
```

Comment:
Probably better to depend on the currentUser.id, right? Or is there any reason to recreate this for any change in the user?

Reply:
corrected